CHANGELOG.md

```diff
@@ -28,6 +28,95 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+<!-- dumped content start -->
+
+## Unreleased
+
+### [shopsys/coding-standards]
+
+#### Added
+
+- [#384] [coding-standards] added new rule to keep class spacing consistent, Thanks to [@TomasVotruba]
+### [shopsys/framework]
+
+#### Changed
+
+- [#295] [framework] javascript compiling: improved memory consumption, Thanks to [@pk16011990]
+
+#### Fixed
+
+- [#386] [framework] fixed FileUpload wrong second parameter for UploadFailedException, Thanks to [@Maybeway]
+- [#420] [framework] Order flow fix - function call on string ([#419]), Thanks to [@jDolba]
+
+#### Removed
+
+- [#428] [framework] *EditFromTypes removed
+### [shopsys/monorepo-tools]
+
+#### Fixed
+
+- [#433] [monorepo-tools] fix: splitting now works correctly on repositories containing files with spaces, Thanks to [@dominikkaluza]
+### [shopsys/project-base]
+
+#### Changed
+
+- [#449] [project-base] config files are now split into individual package configuration files
+- [#447] [project-base] improved UnableToResolveDomainException error page
+
+#### Fixed
+
+- [#431] [project-base]  styling: fixed position of validation error icon for checkboxes, fixes [#142], Thanks to [@kulincz]
+
+#### Unknown Category
+
+- [#432] [project-base] extended Article tests
+### [shopsys/shopsys]
+
+#### Added
+
+- [#409] [shopsys] added framework models extension
+- [#424] [shopsys] adding-a-new-advert-position.md: added instructions for editing images.yml config file
+- [#429] [shopsys] added microservice for product search export
+
+#### Changed
+
+- [#430] [shopsys] Microservices are now built as Docker images
+- [#427] [shopsys] changelog entry for [#420] is now in the correct section and the author is mentioned
+- [#444] [shopsys] Improve Postgres configuration to improve performance
+- [#456] [shopsys] Change recommended stability to dev for installation
+
+#### Fixed
+
+- [#436] [shopsys] fix: Symfony >=3.4.15 marked as conflicting in composer.json
+- [#403] [shopsys] fix inconsistent line separators (changed to \n)
+- [#459] [shopsys] Docker-compose-mac.yml.dist: fix a missing sync external volume
+
+#### Removed
+
+- [#411] [shopsys] docs: native-installation.md: removed information about platform specific sorting from troubleshooting section
+- [#451] [shopsys] remove unnecessary configuration from php.ini files
+
+#### Unknown Category
+
+- [#458] [shopsys] Demoshop experiences documentation
+- [#438] [shopsys] Attribute telephone moved from a billing address to the personal data…
+- [#439] [shopsys] Clarification of Stale bot configuration
+
+## [v7.0.0-alpha5]
```

Comment:
I thought that the linker generates the unreleased changes only :thinking:

Reply:
If you use this package for a repository with 30 tags and empty `CHANGELOG.md`, it will try to cover it all.

All unmentioned PRs are covered.